test/v1/test_account_info.py

```diff
@@ -292,11 +293,48 @@ def test_upgrade_2_default_app_key(self):
     def _make_info(self):
         return self._make_sqlite_account_info()
 
-    def _make_sqlite_account_info(self, last_upgrade_to_run=None):
+    def _make_sqlite_account_info(self, env=None, last_upgrade_to_run=None):
         """
-        Returns a new StoredAccountInfo that has just read the data from the file.
+        Returns a new SqliteAccountInfo that has just read the data from the file.
+
+        :param dict env: Override Environment variables.
         """
-        return SqliteAccountInfo(file_name=self.db_path, last_upgrade_to_run=None)
+        with mock.patch.dict('os.environ', env or {}):
+            return SqliteAccountInfo(
+                file_name=self.db_path if not env else None,
+                env=env or {},
```

Comment:
you have removed the `env` argument from `SqliteAccountInfo` constructor, but it remained here in tests, causing all tests of `SqliteAccountInfo` to fail. Could you please fix it and run `./pre-commit.sh` to make sure all tests pass before we merge it in?

Also, could you please add `Add support for XDG_CONFIG_HOME (0az)` to changelog section in readme?